steward_proto_rust/src/prost/steward.v3.rs

```diff
@@ -2986,6 +2989,19 @@ pub mod cellar_v2_5 {
         #[prost(string, tag = "1")]
         pub new_cap: ::prost::alloc::string::String,
     }
+    ///
+    /// Allows strategist to set the share price oracle contract
+    ///
+    /// Represents function `setSharePriceOracle(uint256, address)`
+    #[derive(serde::Deserialize, serde::Serialize, Clone, PartialEq, ::prost::Message)]
+    pub struct SetSharePriceOracle {
```

Comment:
I feel like I'm missing something here, I see the proto binding for SetSharePriceOracle but where is the generated encoding function in the ABI bindings?

Reply:
Oh, I guess it was in the base cellar ABI we already used, and we had just never touched it. I thought we had talked about having to use a different source ABI but it's there after all.

Reply:
A previous 2.5 ABI we generated happened to have it